src/scss/components/_home.scss

```diff
@@ -0,0 +1,50 @@
+@import '../variables';
+
+.home{
+  height: 75vh;
+  width: 55vw;
+  margin: 50px auto;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: center;
+  background-color: $middle;
```

Comment:
middle is really nice name for color ... i guess in is violet , or green ... can you use better naming ?